src/navigation/Nav.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { v4 as uuid } from 'uuid';
+import logo from '../images/planet.png';
+
+export default function Nav() {
+  const routLinks = [
+    {
+      path: '/',
+      text: 'Rockets',
+    },
+    {
+      path: '/mission',
+      text: 'Missions',
+    },
+    {
+      path: '/profile',
+      text: 'My Profile',
+    },
```

Comment:
- [ ] [REQUIRED] Only the active page should be indicated with an underline. Kindly implement this.